docs/source/workbench/plotwindow.rst

```diff
@@ -31,12 +31,13 @@ Plot Window Context Menu
 
 Right-clicking within your plot will open the plot context menu.
 From this menu you can:
-- set the axes' scales to linear or logarithmic.
```

Comment:
Sorry to be picky. We need a single blank line above this line or the list is formatted as pre-formatted block (note the line down the left-hand side):
![Screenshot from 2019-08-23 16-22-53](https://user-images.githubusercontent.com/733773/63604277-3df0da00-c5c3-11e9-935c-53dc1a19e2fd.png)